src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
How close do we thing we are to doing a pull request?  Another
approach would be to create a topic branch in the main repo.
I am getting pressure to genome annotations more visibility.

Mark